net-mail/notmuch/notmuch-0.30.ebuild

```diff
@@ -0,0 +1,212 @@
+		virtual/perl-Pod-Parser
+	)
+	nmbug? ( dev-vcs/git )
+"
+
+DOCS=( AUTHORS NEWS README )
+DOC_CONTENTS="There are a few backward-incompatible changes between
+notmuch version 0.18 < x <= 0.18. Please consult the NEWS file (0.18
+section) before first use."
+MY_LD_LIBRARY_PATH="${WORKDIR}/${P}/lib"
+SITEFILE="50${PN}-gentoo.el"
+
+bindings() {
+	local rc=0
+	if use python; then
+		pushd bindings/python-cffi || die
+		"$@"
+		rc=$?
+		popd || die
+	fi
+	return $rc
+}
+
+pkg_setup() {
+	use emacs && elisp-check-emacs-version
+}
+
+src_unpack() {
+	unpack "${P}".tar.xz
+	if use test; then
+		mkdir -p "${S}"/test/test-databases || die
+		cp "${DISTDIR}"/database-v1.tar.xz "${S}"/test/test-databases/ || die
+	fi
+}
+
+src_prepare() {
+	local _deps=""
+	default
+
+	# Python bindings
+	bindings distutils-r1_src_prepare
+	mv contrib/notmuch-mutt/README contrib/notmuch-mutt/README-mutt || die
+
+	# Ensure that a new Makefile.config will be generated
+	rm -f Makefile.config || die
+
+	# Override dependencies for 'install' target
+	use doc && _deps="install-info"
+	sed -e "s/^install:.\+/install: all ${_deps}/" -i Makefile.local || die
+
+	if use test; then
+		append-cflags -g
+		append-cxxflags -g
+	fi
+
+	if [[ ${CHOST} == *-solaris* ]] ; then
+		# Non-autoconf configure
+		append-ldflags -lnsl -lsocket
+	fi
+}
+
+src_configure() {
+	python_setup # For sphinx
+	local _args=(
+		--bashcompletiondir="$(get_bashcompdir)"
+		--emacslispdir="${EPREFIX}/${SITELISP}/${PN}"
+		--emacsetcdir="${EPREFIX}/${SITEETC}/${PN}"
+		--without-desktop
+		--without-ruby
+		--zshcompletiondir="${EPREFIX}/usr/share/zsh/site-functions"
+		$(use_with emacs)
+		$(use_with doc api-docs)
+	)
+	tc-export CC CXX
+	econf "${_args[@]}"
+}
+
+src_compile() {
+	python_setup # For sphinx
+	V=1 default
+	bindings distutils-r1_src_compile
+
+	if use mutt; then
+		pushd contrib/notmuch-mutt || die
+		emake notmuch-mutt.1
+		popd || die
+	fi
+}
+
+src_test() {
+	pax-mark -m notmuch
+	LD_LIBRARY_PATH="${MY_LD_LIBRARY_PATH}" V=1 default
+	pax-mark -ze notmuch
+}
+
+src_install() {
+	default
+
+	if use doc; then
+		doman doc/_build/man/man?/*.?
+	fi
+
+	if use emacs; then
+		elisp-site-file-install "${FILESDIR}/${SITEFILE}" || die
+	fi
+
+	if use nmbug; then
+		dobin devel/nmbug/nmbug
+		dobin devel/nmbug/notmuch-report
+	fi
+
+	if use mutt; then
+		pushd contrib/notmuch-mutt || die
+		dobin notmuch-mutt
+		doman notmuch-mutt.1
+		insinto /etc/mutt
+		doins notmuch-mutt.rc
+		dodoc README-mutt
+		popd || die
+	fi
+
+	DOCS="" bindings distutils-r1_src_install
+}
+
+pkg_preinst() {
+	local _rcfile="/etc/mutt/notmuch-mutt.rc"
+	if use mutt && ! [[ -e ${ROOT}${_rcfile} ]]; then
+		elog "To enable notmuch support in mutt, add the following line"
+		elog "to your mutt config file:"
+		elog ""
+		elog "  source ${_rcfile}"
+	fi
+}
+
+pkg_postinst() {
+	use emacs && elisp-site-regen
+	if has_version '<net-mail/notmuch-0.18'; then
+		FORCE_PRINT_ELOG=1 readme.gentoo_print_elog
+	fi
```

Comment:
This looks redundant honestly, allows you to remove the eclass inherit and `DOC_CONTENT` variable here.